doc/mgr/dashboard.rst

```diff
@@ -181,13 +181,16 @@ app.
 Username and password
 ^^^^^^^^^^^^^^^^^^^^^
 
-In order to be able to log in, you need to define a username and password, which
-will be stored in the MON's configuration database::
+In order to be able to log in, you need to create an user account and associate
```

Comment:
`s/an user/a user/`

Reply:
Good spotting :)

https://english.stackexchange.com/questions/4700/is-it-a-uniform-or-an-uniform